Duplicati/Library/Main/Operation/ListControlFilesHandler.cs

```diff
@@ -68,7 +68,7 @@ public void Run(IEnumerable<string> filterstrings = null, Library.Utility.IFilte
                                     if (Library.Utility.FilterExpression.Matches(filter, cf.Key))
                                         files.Add(new ListResultFile(cf.Key, null));
                             
-                            m_result.SetResult(new Library.Interface.IListResultFileset[] { new ListResultFileset(fileversion.Key, fileversion.Value.Time, -1, -1) }, files);
+                            m_result.SetResult(new Library.Interface.IListResultFileset[] { new ListResultFileset(fileversion.Key, 0, fileversion.Value.Time, -1, -1) }, files);

```

Comment:
What's the meaning of `0` here?  Is it meant to represent `BackupType.PARTIAL_BACKUP`?

Reply:
Correct and now fixed.